python/app/console.py

```diff
@@ -16,9 +16,15 @@
 import sgtk
 from sgtk.errors import TankError
 from sgtk.platform.engine import current_engine
-from sgtk.platform.qt import QtCore, QtGui
+try:
```

Comment:
I think we should separate the `ShotgunPythonConsoleWidget` into a separate file so that this file has no SG dependencies. Then flame can import this file directly. And we don't have to worry about the imports or settings.